src/PeriodEnd/CoInvestedPayments/SFA.DAS.Payments.Calc.CoInvestedPayments/CoInvestedPaymentsProcessor.cs

```diff
@@ -61,15 +60,7 @@ public virtual void Process()
 
                     foreach (var paymentDue in providerPaymentsDueQueryResponse.Items)
                     {
-                        if (paymentDue.AmountDue > 0)
-                        {
-                            AddCoInvestedPaymentsForLearner(learnerLevelPaymentsForProvider, collectionPeriod, paymentDue);
-                        }
-                        else if (paymentDue.AmountDue < 0)
-                        {
-                            MakeCoInvestedRefundsForLearner(learnerLevelPaymentsForProvider, collectionPeriod, paymentDue);
-
-                        }
+                        AddCoInvestedPaymentsForLearner(learnerLevelPaymentsForProvider, collectionPeriod, paymentDue);
```

Comment:
Is it worth putting a check in to ensure we are only processing +ve values. Or is this done when getting records from the respository